doc/source/groupby.rst

```diff
@@ -497,6 +497,28 @@ index are the group names and whose values are the sizes of each group.
 
    ``nth`` can act as a reducer *or* a filter, see :ref:`here <groupby.nth>`
 
+   Decimal columns are "nuisance" columns that .agg automatically excludes in groupby.
```

Comment:
The word "nuisance" comes off a little too strong IMO.  Just state that they're excluded and (very briefly) why.

Reply:
no, we call excluded columns ``nuisance`` so this is ok.